docs/zkapps/tutorials/09-recursion.mdx

```diff
@@ -317,4 +412,4 @@ You learned about:
 
 Recursive zero knowledge proofs can help you build powerful zkApps.
 
-Check out [Tutorial 10: Account Updates](/zkapps/tutorials/account-updates) to learn about account updates, the underlying structure of zkApps, and how they enable permissions, preconditions, and composability.
+Check out [Tutorial 10: Account Updates](/zkapps/tutorials/account-updates) to learn about account updates, the underlying structure of zkApps, and how they enable permissions, preconditions, and composability.
```

Comment:
@LuffySama-Dev  for some reason, this link is breaking Vercel CI

Do you have the entire docs2 repo cloned for this fork? 

Tip: See https://github.com/o1-labs/docs2/blob/main/README.md#how-to-see-your-local-doc-updates-during-development 

and https://github.com/o1-labs/docs2/blob/main/README.md#debugging-build-errors 

This link is valid, but the PR cannot move forward with failing CI.

Reply:
Okay, I will check and update here before EOD.

Reply:
Hi @barriebyron ,

I tried to check this locally and link is fine, but the build is failing.
I am checking why is it happening as only line number changed an rest everything is same.

Thank you

Reply:
Hi @barriebyron ,

It seems in one of the recent PR #839 , a extra `(` was added in `on-chain-values.mdx`.
It was creating issues for us during build.
I have removed the extra parenthesis and build is successful locally.
I am committing it right away.

![Screenshot 2024-02-13 at 11 38 54 PM](https://github.com/o1-labs/docs2/assets/55076843/dd3265a6-1772-4633-b8a8-041a539e8b1c)



Thank You 😃

Reply:
yes you're right, per usual! thank you, fixed on main